build.sbt

```diff
@@ -133,9 +133,9 @@ lazy val sdkSpring = project
     buildInfoPackage := "kalix.springsdk",
     Compile / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Server),
     Compile / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala),
-    Test / javacOptions ++= Seq("-parameters", "--release", "17"), // for Jackson
-    Test / scalacOptions ++= Seq("-release", "17"),
-    IntegrationTest / javacOptions += "-parameters", // for Jackson
+    Compile / javacOptions ++= Seq("-parameters" /* for Jackson */, "--release", "17"),
+    Compile / scalacOptions ++= Seq("-release", "17"),
```

Comment:
Actually we should revert this. We only want the tests to be on 17, not the production artifacts.